dev/src/main/resources/messages/ui/CodewindUIBundle.properties

```diff
@@ -243,8 +243,10 @@ DialogYesButton=Yes
 DialogNoButton=No
 DialogCancelButton=Cancel
 
-ProjectNotImportedDialogTitle=The {0} project is not imported into Eclipse
-ProjectNotImportedDialogMsg=There is no source available for debugging because the project has not been imported. Import the {0} project?
+ProjectMustBeOpenedToDebugTitle=Project Must Be Opened
+ProjectMustBeOpenedToDebugMessage=To perform the selected action, open the project in a window. Continue?
+ProjectIsOpenedInAnotherWindowTitle=Select Existing Window
+ProjectIsOpenedInAnotherWindowMsg=The project was opened in another window.  Select it and continue?
```

Comment:
@keithchong Please remove the extra space after period in line 249. Thank you!

Reply:
Done.